app/ante/evm/ante_test.go

```diff
@@ -40,6 +40,25 @@ func (suite *AnteTestSuite) TestAnteHandler() {
 		suite.app.FeeMarketKeeper.SetBaseFee(suite.ctx, big.NewInt(100))
 	}
 
+	ethTxContractParams := &evmtypes.EvmTxArgs{
```

Comment:
Rename to `ethContractCreationTxParams`, otherwise at a glance, it's unclear whether this is to create or interact with a contract.